tesoro/utils.py

```diff
@@ -6,14 +6,15 @@
 logger = logging.getLogger(__name__)
 
 
-def kapicorp_labels(req_obj):
+def kapicorp_labels(req_uid, req_obj):
     "returns kapicorp labels dict for req_obj"
     labels = {}
     try:
         for label_key, label_value in req_obj["metadata"]["labels"].items():
             if label_key.startswith("tesoro.kapicorp.com"):
                 labels[label_key] = label_value
     except KeyError:
+        logger.error("request_id=%s Tesoro label not found", req_uid)
```

Comment:
This should be debug, not error. Otherwise it will spam logging with objects that don't set the tesoro label